doc/design/active-audio-selecting.md

```diff
@@ -0,0 +1,24 @@
+| OWT Active Audio Stream Introduction |
+
+
+# Overview
+
+OWT conference provide functionality to select K (K = 3 by default) most active audio streams in the conference.
+
+# How to enable
+The active audio selecting feature can be enabled by setting `selectActiveAudio` to true in room configuration. When it's enabled, an audio node will be used for selecting audio streams for each room, and K audio-only streams will be produced as most active audio streams in the conference.
+
+# Stream
```

Comment:
highlighting-code-blocks?

https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks